### PR TITLE
Add note about creating entry files in JS and Webpack

### PR DESCRIPTION
**Pull request by scottcorgan**
🛑 Blocked by https://github.com/desiringgod/www/issues/3166 🛑  (Need to probably put this in `www`)

---

Documents https://github.com/desiringgod/www/pull/3189

## TODO

- [ ] Note where module files should still live per https://github.com/desiringgod/guides/pull/51#discussion_r136351339

### CODE REVIEW

**Comment by scottcorgan** [comment]
@hchasey assigning to you because it won't let me assign to engineering

**Comment by scottcorgan** [comment]
Now that I think about this, it is `www` specific. So, maybe this show go over there.

I'm going to mark it as blocked by https://github.com/desiringgod/www/issues/3166 because I don't really want to put it into the wiki.

**Comment by hchasey** [comment]
Thanks @scottcorgan – I'll look at the wiki issue and make some updates....

File: style/javascript/README.md [hunk 1]
```diff
@@ -25,3 +25,19 @@ JavaScript
...
+
+Our build system requires what's called an entry file. An entry file is the root file which imports all the code used by that particular script.
+An example of this is our `application_development.js` entry file. If you open that file, you'll see a bunch of imports and
+instantiations. Entry files should only contain code that gets things going. Try to avoid putting logic into these files.
+
+Every now and the, you'll want to write some JavaScript that is particular to a single page, so it might make sense to
+only include that JavaScript on that one page. Here is how you would go about doing that.
+
+1. Create an entry file in the `app/client/webpack_entry_files/javascript` directory.
+2. Import any modules that will be used for this page.
```
Comment (pjmelling) [inline style/javascript/README.md]: @scottcorgan and the place to put the module is still in app/client/javascript/modules?

Comment (scottcorgan) [reply style/javascript/README.md]: @pjmelling yep! I should probably note that as well. Thanks for the heads up.

**Comment by hchasey** [comment]
@scottcorgan – is this unblocked now due to https://github.com/desiringgod/www/issues/3195? Looking at this doc, it seems like this might fit better in www, but we can discuss.

**Comment by scottcorgan** [comment]
@hchasey yep, I'll move this to www 👍

**Pull request closed without merging**